app/src/main/java/bg/zahov/app/util/Extensions.kt

```diff
@@ -252,14 +252,14 @@ fun Sets.toRealmSets(): RealmSets {
 }
 
 fun RealmExercise.toExercise(): Exercise? {
-    return if (this.name != "default" && BodyPart.fromKey(this.bodyPart) != null && Category.fromKey(
-            this.category
-        ) != null
-    ) {
+    val bodyPart = BodyPart.entries.firstOrNull { it.key == this.bodyPart }
+    val category = Category.entries.firstOrNull { it.key.equals(this.category, true) }
+
+    return if (this.name != "default" && bodyPart != null && category != null) {
```

Comment:
Maybe use `SetType.DEFAULT` instead of "default"?

Reply:
I am not sure what name = "default is " and if i use SetType.DEFAULT will be appropriate

Reply:
If you think `SetType` is not the correct one it's best to extract the "default" string into a constant and reuse it wherever this check is made. That will give more context on why the check is there in the first place.

Reply:
done